Headers/AppKit/NSSeguePerforming.h

```diff
@@ -0,0 +1,47 @@
+/* Definition of class NSSequePerforming
```

Comment:
You seem to have renamed the file but not the class in it and all the other places where the wrong name gets used.

Reply:
Ah, okay, sorry about that. :)